README.rst

```diff
@@ -140,7 +140,23 @@ Skipping Tests
 ~~~~~~~~~~~~~~
 
 Doctest provides the ``+SKIP`` directive for skipping statements that should
-not be executed when testing documentation. However, it is often useful to be
+not be executed when testing documentation.
+
+.. code-block:: python
+
+    >>> open('file.txt') # doctest: +SKIP
+
+And in `.rst` documentation, whole code example blocks can be skipped with the
+directive
+
+.. code-block:: rst
```

Comment:
this is not rst codeblock though, but python. right?

Reply:
This is intended to show code examples within sphinx `rst` files.  But yeah, the actual block is in python.  If labelled `rst` the `doctest-skip` gets highlighted in blue. But if labelled `python`, it does not, and the code itself is syntax highlighted.  Preference?

Reply:
Ohh, so `code-block` is for syntax highlighting only. I learnt something new today. 
This case your original is indeed better, I'm sorry for the noise.

Reply:
I believe `.. code-block::` will render a code block in RST, but the highlighting is controlled by the subsequent directive `python`, `ini`, `rst`, etc.?  Of course there's other ways to signal a code block via `::` ending the paragraph before, or just indentation.

I'm learning this stuff as I go along as well, so hence my desire to see it documented somewhere.